test/integration/sequelize.test.js

```diff
@@ -12,7 +12,7 @@ const sinon = require('sinon');
 const current = Support.sequelize;
 
 const qq = str => {
-  if (dialect === 'postgres' || dialect === 'mssql') {
+  if (dialect === 'postgres' || dialect === 'mssql' || dialect === 'db2') {
```

Comment:
```suggestion
  if (['postgres', 'mssql', 'db2'].includes(dialect)') {
```

Reply:
Again my bad. Corrected. A similar change needs to be done in line 18 but I am not touching it as it is not a db2 change.